deploy/runtools/run_farm_deploy_managers.py

```diff
@@ -94,6 +142,12 @@ def terminate_instance(self) -> None:
         """
         raise NotImplementedError
 
+    @classmethod
+    @abc.abstractmethod
+    def sim_command_requires_sudo(self) -> bool:
```

Comment:
nit: if this needs to be a `@classmethod` it's customary to have the param be called `class` instead of `self` because that's what you'll be getting.